// cronofy/build/index.js
'use strict';

Object.defineProperty(exports, '__esModule', {
  value: true
});

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { 'default': obj }; }

var _methodsAccountInformation = require('./methods/account-information');

var _methodsAccountInformation2 = _interopRequireDefault(_methodsAccountInformation);

var _methodsCreateEvent = require('./methods/create-event');

var _methodsCreateEvent2 = _interopRequireDefault(_methodsCreateEvent);

var _methodsCreateNotificationChannel = require('./methods/create-notification-channel');

var _methodsCreateNotificationChannel2 = _interopRequireDefault(_methodsCreateNotificationChannel);

var _methodsDeleteEvent = require('./methods/delete-event');

var _methodsDeleteEvent2 = _interopRequireDefault(_methodsDeleteEvent);

var _methodsDeleteNotificationChannel = require('./methods/delete-notification-channel');

var _methodsDeleteNotificationChannel2 = _interopRequireDefault(_methodsDeleteNotificationChannel);

var _methodsFreeBusy = require('./methods/free-busy');

var _methodsFreeBusy2 = _interopRequireDefault(_methodsFreeBusy);

var _methodsListCalendars = require('./methods/list-calendars');

var _methodsListCalendars2 = _interopRequireDefault(_methodsListCalendars);

var _methodsListNotificationChannels = require('./methods/list-notification-channels');

var _methodsListNotificationChannels2 = _interopRequireDefault(_methodsListNotificationChannels);

var _methodsProfileInformation = require('./methods/profile-information');

var _methodsProfileInformation2 = _interopRequireDefault(_methodsProfileInformation);

var _methodsReadEvents = require('./methods/read-events');

var _methodsReadEvents2 = _interopRequireDefault(_methodsReadEvents);

var _methodsRefreshAccessToken = require('./methods/refresh-access-token');

var _methodsRefreshAccessToken2 = _interopRequireDefault(_methodsRefreshAccessToken);

var _methodsRequestAccessToken = require('./methods/request-access-token');

var _methodsRequestAccessToken2 = _interopRequireDefault(_methodsRequestAccessToken);

var _methodsRevokeAuthorization = require('./methods/revoke-authorization');

var _methodsRevokeAuthorization2 = _interopRequireDefault(_methodsRevokeAuthorization);

var methods = {
  accountInformation: _methodsAccountInformation2['default'],
  createEvent: _methodsCreateEvent2['default'],
  createNotificationChannel: _methodsCreateNotificationChannel2['default'],
  deleteEvent: _methodsDeleteEvent2['default'],
  deleteNotificationChannel: _methodsDeleteNotificationChannel2['default'],
  freeBusy: _methodsFreeBusy2['default'],
  listCalendars: _methodsListCalendars2['default'],
  listNotificationChannels: _methodsListNotificationChannels2['default'],
  profileInformation: _methodsProfileInformation2['default'],
  readEvents: _methodsReadEvents2['default'],
  refreshAccessToken: _methodsRefreshAccessToken2['default'],
  requestAccessToken: _methodsRequestAccessToken2['default'],
  revokeAuthorization: _methodsRevokeAuthorization2['default']
};

exports['default'] = methods;
module.exports = exports['default'];
