// cronofy/build/methods/free-busy.js
'use strict';

Object.defineProperty(exports, '__esModule', {
  value: true
});

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { 'default': obj }; }

var _lodash = require('lodash');

var _lodash2 = _interopRequireDefault(_lodash);

var _whenNode = require('when/node');

var _whenNode2 = _interopRequireDefault(_whenNode);

var _origami = require('origami');

var _libRestClient = require('../lib/rest-client');

var _libRestClient2 = _interopRequireDefault(_libRestClient);

function freeBusy(options, callback) {
  var settings = {
    method: 'GET',
    path: 'https://api.cronofy.com/v1/free_busy',
    headers: {
      Authorization: 'Bearer ' + options.access_token
    },
    params: _lodash2['default'].omit(options, 'access_token')
  };
  var result = (0, _libRestClient2['default'])(settings).fold(_origami.reach, 'entity');

  return _whenNode2['default'].bindCallback(result, callback);
}

exports['default'] = freeBusy;
module.exports = exports['default'];
